### PR TITLE
Updated and refactored.

### PR DESCRIPTION
**Pull request by codingdash**
This is simple simulation of how a popup system can be implemented in NodeJs which can be triggered using popup configuration of Exotel IVR connect applet.

### CODE REVIEW

**Comment by codingdash** [comment]
Updated java examples. Now it has few improvements over the past version-
1. Works fine with JRE 1.5 or greater
2. Refactored

**Comment by codingdash** [comment]
Updated java examples. Now it has few improvements over the past version-
1. Works fine with JRE 1.5 or greater
2. Refactored

File: Java_Samples/ConnectCustomerToFlow.java [hunk 1]
```diff
@@ -1,99 +1,58 @@
...
+        //Update here
+        String exotelSid = "<ExotelSid>";
+        String token = "<Token>";
+        String calltype = "<CallType>";
+        String appId = "<AppId>";
+        String customerNumber = "<CustomerNumber>";
+        String exophone = "<Exophone-number>";
+
+        String authStr = exotelSid + ":" + token;
+        String url = "https://" + authStr +
```
Comment (sarathsp06) [inline Java_Samples/ConnectCustomerToFlow.java]: Why  doing both setting header and username password combination in URL ? .Do only one ,ie, preferably set 
post.setHeader("Authorization", "Basic " + authStringEnc);
and remove sid:toeken  from url

File: README.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
-ExotelAPI
-=========
+# [Exotel](http://exotel.in/)
 
-Exotel's Call and SMS API sample code
\ No newline at end of file
+Exotel is a software, which can be used as a call center in your business. It doesn't require any
```
Comment (sarathsp06) [inline README.md]: Exotel is a software ??

**Pull request closed without merging**